src/test_utils.jl

```diff
@@ -765,15 +765,22 @@ function functions_for_mutable_structs()
 end
 
 """
-    test_rule_and_type_interactions(rng::AbstractRNG, x)
+    test_rule_and_type_interactions(rng::AbstractRNG, p)
```

Comment:
(not for this line) in the first sentence of the following doc, there is an `x` possibly should be `p` (also the first sentence of the second paragraph)?

Reply:
Good catch, thanks!